manifests/b/BenHollis/PNGGauntlet/3.1.1/BenHollis.PNGGauntlet.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: BenHollis.PNGGauntlet
+PackageVersion: 3.1.1
+PackageLocale: en-US
+Publisher: Ben Hollis
+PackageName: PNGGauntlet
+License: Copyright (c) 2012 Benjamin Hollis

```

A copyright is not a license
```suggestion
License: Proprietary
Copyright: Copyright (c) 2012 Benjamin Hollis
```